.travis.yml

```diff
@@ -24,60 +15,68 @@ stages:
 
 jobs:
   include:
-    - name: "Linters"
+    - name: 'Linters'
       stage: test
       language: python
-      python: "3.6"
+      python: '3.6'
       install:
         - pip3 install --upgrade pycodestyle
       script:
         - pycodestyle --max-line-length=99 --exclude='./python/lookout/sdk' .
 
-    - name: "Python: example integration tests"
+    - name: 'Python: example integration tests'
       stage: test
       language: python
-      python: "3.6"
+      python: '3.6'
       install:
+        - ./_tools/ci-run-bblfsh.sh
         - pip3 install -e python
       script:
         - ./_tools/install-lookout-latest.sh
-        - (python3 language-analyzer.py |& tee -a ./py-analyzer.log)&
-        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2021"
-      after_failure:
-        - echo "test"
-        - cat ./py-analyzer.log
-        - cat ./lookout-install.log
+        - (python3 -u language-analyzer.py |& tee ./py-analyzer.log)&
+        - sleep 5s
+        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2021" |& tee ./sdk.log
+        - grep -v -i "error" ./py-analyzer.log > /dev/null
+        - grep -v -i "error" ./sdk.log > /dev/null
 
-    - name: "Golang: example integration tests"
+    - name: 'Golang: example integration tests'
       stage: test
       language: go
-      go: "1.10"
+      go: '1.10'
       install:
         - go version
         - go get .
       script:
+        - ./_tools/ci-run-bblfsh.sh
         - ./_tools/install-lookout-latest.sh
-        - (go run language-analyzer.go |& tee -a ./go-analyzer.log)&
-        - sleep 1s
-        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2020"
-      after_failure:
-        - cat ./go-analyzer.log
-        - cat ./lookout-install.log
+        - (go run language-analyzer.go |& tee ./go-analyzer.log)&
+        - sleep 5s
+        - ./lookout-sdk review --log-level=debug --from 0a9d1d159d2b0064c32df8d2287b174a91390b1a --to HEAD "ipv4://localhost:2020" |& tee ./sdk.log
```

Comment:
In my case, it's failing because 
```
[2018-11-21T16:36:47.239263534+01:00]  INFO analyzing '.github/CODE_OF_CONDUCT.md' in Markdown
language-analyzer: uast.cc:268: Nodes* UastFilter(const Uast*, void*, const char*): Assertion `node' failed.
SIGABRT: abort
```
what I think it is wrong.

If I try with `--from 103fb7776d4c20bcefaf87a66f65e1168454b0d8 --to 63695279e311cd34a5b7ad473cc2063ba4334dda` it suceedes

Reply:
Please explain how did you get this error.
If example python analyzer fails on `md` files we need to fix analyzer.
It's an example that supposed to be copy-pasted better provide code without bugs for copy-past.

P.S. Better fix it in separate pr.

Reply:
In my HEAD I modified `.md` files, so it failed.

For sure, if we fix the analyzer the integration test will pass, but I'm not sure that the purpose of the integration test is to ensure that the example analyzer is ok, but to ensure all the pieces are properly joined.

My point is that the integration test should not rely on moving commits (that sometimes can fail), but in fixed ones to ensure that all pieces are also working properly

Reply:
I prefer the current approach as long as it doesn't cause problems.
Otherwise, we would never find this bug for example.